Makefile

```diff
@@ -624,6 +625,9 @@ $(BUILD_DIR)/initmod.%.bc: $(BUILD_DIR)/initmod.%.ll $(BUILD_DIR)/llvm_ok
 $(BUILD_DIR)/initmod.%.cpp: $(BIN_DIR)/bitcode2cpp $(BUILD_DIR)/initmod.%.bc
 	./$(BIN_DIR)/bitcode2cpp $* < $(BUILD_DIR)/initmod.$*.bc > $@
 
+$(BUILD_DIR)/initmod.declarations.cpp: $(BIN_DIR)/bitcode2cpp $(SRC_DIR)/runtime/HalideRuntime.h
+	./$(BIN_DIR)/bitcode2cpp declarations < $(SRC_DIR)/runtime/HalideRuntime.h > $@
```

Comment:
We should consider renaming bitcode2cpp, or at least add a comment here as to what is going on. (I'm guessing the text of HalideRuntime.h will be put into generated header files, but I haven't read the rest of the code yet.)